GeometryService/inc/STMMaker.hh

```diff
@@ -73,13 +76,20 @@ namespace mu2e {
     double      _FOVCollimatorHole1RadiusDnStr;
     bool        _FOVCollimatorHole1LinerBuild;
     double      _FOVCollimatorHole1LinerThickness;
+    bool        _FOVCollimatorHole1Liner2Build;//wyq useless
```

Comment:
Why is this marked useless? It appears to be used in the collimator constructor and when creating it in Geant4